src/msal-client-credentials-flow/Program.cs

```diff
@@ -6,6 +7,11 @@
 
 // <ms_docref_add_msal>
 WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
+// Acquire a token from Azure AD for this client to access Microsoft Graph based

```

Comment:
Sorry, channeling my inner Marsh (per his request)

```suggestion
// Acquire an access token from Azure AD for this client to access Microsoft Graph based
```